### ISSUE TITLE
css() isn't picking up pseudo-selector styles defined in emotion css prop

### ISSUE THREAD

**Issue by kevinSuttle** [styled-system/extras#19]
Is there something I'm missing?

```jsx
css={css({':focus': {borderColor: 'blue20'}}
```
<img width="373" alt="Image 2019-05-03 at 11 25 45 AM" src="https://user-images.githubusercontent.com/95672/57147714-35676000-6d96-11e9-945f-cb15f02185f4.png">

This is being used on a `<Flexbox />` component built in `@emotion/styled` + `styled-system`.

```js
// @flow

import styled from '@emotion/styled';

import {
  alignItems,
  alignContent,
  justifyContent,
  justifyItems,
  flexWrap,
  flexBasis,
  flexDirection,
  flex,
  justifySelf,
  alignSelf,
  order,
} from 'styled-system';
import shouldForwardProp from '@styled-system/should-forward-prop';

import {Box} from './Box';

export const Flexbox = styled(Box, {
  shouldForwardProp,
})(
  {
    boxSizing: 'border-box',
    display: 'flex',
    label: 'flexbox-primitive',
  },
  alignItems,
  alignContent,
  justifyContent,
  justifyItems,
  flexWrap,
  flexBasis,
  flexDirection,
  flex,
  justifySelf,
  alignSelf,
  order
);

```

What's odd is that I can get it to work here: https://stackblitz.com/edit/react-zzznyy

**Comment by jxnblk** [comment]
What version of `@styled-system/css` are you using?

**Comment by kevinSuttle** [comment]
`2.0.0.`

Actually, I think this might have been due to not having a border-width or style set.

**Comment by kevinSuttle** [comment]
~~`2.0.0.`~~

~~Actually, I think this might have been due to not having a border-width or style set.~~

Edit: `2.0.0` fixes this. I misspoke. It's broken in `1.0.3`.

**Comment by jxnblk** [comment]
Yeah, v2 should support `borderColor` (v1 does not) – I'd also check to make sure theme context is being passed through when you see things like this

**Comment by kevinSuttle** [comment]
👍 

Aside: I rolled back the version in the demo https://stackblitz.com/edit/react-zzznyy and it's showing some odd behavior. It's applying the focus color right away. 

<img width="1404" alt="Image 2019-05-03 at 12 35 48 PM" src="https://user-images.githubusercontent.com/95672/57151868-02c26500-6da0-11e9-8cf4-dc53e91b84a6.png">
